eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/client/EclairRpcClient.scala

```diff
@@ -560,22 +561,42 @@ class EclairRpcClient(val instance: EclairInstance)(
 
   private var process: Option[Process] = None
 
-  def start(): Unit = {
+  def start(): Future[Unit] = {
 
-    if (process.isEmpty) {
-      val p = Process(
-        s"java -jar -Declair.datadir=${instance.authCredentials.datadir.get} $pathToEclairJar &")
-      val result = p.run()
-      logger.info(
-        s"Starting eclair with datadir ${instance.authCredentials.datadir.get}")
+    val _ = {
+      if (process.isEmpty) {
+        val p = Process(
+          s"java -jar -Declair.datadir=${instance.authCredentials.datadir.get} $pathToEclairJar &")
```

Comment:
The error message that potentially triggers on `.get` is somewhat cryptic. Would it be better to  add a `require` statement or something similar on the line above?